docs/exts/removemarktransform.py

```diff
@@ -44,7 +44,7 @@ def apply(self, **kwargs):
         for node in self.document.traverse(nodes.literal_block):
             if self.is_pycode(node):
                 source = node.rawsource
-                source = docmark_re.sub("", source)
+                source = docmark_re.sub(r"", source)
```

Comment:
Why the `r`?

Reply:
Removed the r, in the next commit. Was a syntax error.